Sm0k3s/SendIT
# #162002522 Builds endpoint for user login

#### What does this PR do?
Creates endpoint to login users for the api v2
#### Description of Task to be completed?
Finish endpoint to login users
```POST /api/v2/auth/login  ```
#### How should this be manually tested?
Clone this repo and create a python3 virtualenv 
```bash
$ python3 -m venv venv
$ source venv/bin/activate
```
After activating the environment, install the dependencies 
```
$ pip install -r requirements.txt
Open postman and test the endpoint 
```POST /api/v2/auth/login ```
The expect payloads is ```   
       {"username":"sm0k3s",
	"password":"******"}
```
#### What are the relevant pivotal tracker stories?
[#162002522](https://www.pivotaltracker.com/n/projects/2213547/stories/162002522)
